fe/src/main/java/org/apache/doris/analysis/InsertStmt.java

```diff
@@ -478,6 +489,24 @@ private void checkHllCompatibility(Column col, Expr expr) throws AnalysisExcepti
         }
     }
 
+    private void checkBitmapCompatibility(Column col, Expr expr) throws AnalysisException {
+        final String bitmapMismatchLog = "Column's agg type is bitmap_union,"
+                + " SelectList must contains bitmap_union column or to_bitmap function's result, column=" + col.getName();
+        if (expr instanceof SlotRef) {
+            final SlotRef slot = (SlotRef) expr;
+            if (slot.getDesc().getColumn().getAggregationType() != AggregateType.BITMAP_UNION ) {
+                throw new AnalysisException(bitmapMismatchLog);
+            }
+        } else if (expr instanceof FunctionCallExpr) {
+            final FunctionCallExpr functionExpr = (FunctionCallExpr) expr;
+            if (!functionExpr.getFnName().getFunction().equalsIgnoreCase(FunctionSet.TO_BITMAP)) {
```

Comment:
I think BITMAP_UNION() is OK too.

And you should add in 
fe/src/main/java/org/apache/doris/planner/BrokerScanNode.java:452
fe/src/main/java/org/apache/doris/planner/StreamLoadScanNode.java:278

Reply:
Update